SU-HSDO/stanford_mrc
# MRCD8-275 Make the menu block title a link

# READY FOR REVIEW
# Summary
- Made the menu block subnav title a link to the root item.

# Needed By (Date)
- End of sprint

# Urgency
- low

# Steps to Test

1. Checkout branch
2. clear cache
3. view any sub menu item and verify the block title on the left is a linked label to the root item.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
